test/reducers/index.spec.tsx

```diff
@@ -1,11 +1,20 @@
+import { ProfileActions } from "../../src/actions/profileActions";
 import { rootReducer } from "../../src/reducers";
+import { IProfile, UserMode } from "../../src/reducers/profileReducer";
+import { RootState } from "../../src/state";
 
 describe("root reducer", () => {
   it("returns the same state as provided", () => {
```

Comment:
The description could be more clear. Currently we have that a rootReducer with an initial state, which is told to set the user to a tenant. The test passes if nothing changes, but we would like to validate that the rootReducer actually makes the changes right?